sdk/monitor/Azure.Monitor.Query/src/Models/ResourceIdList.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.Monitor.Query.Models
+{
+    /// <summary> The comma separated list of resource IDs to query metrics for. </summary>
+    public partial class ResourceIdList
+    {
+        /// <summary> Initializes a new instance of ResourceIdList. </summary>
+        public ResourceIdList(List<string> resourceIds)
```

Comment:
Why not just use the generated constructor?

Reply:
The generated constructor doesn't have a setter that takes in resourceIds so I added this method as an overload.